compiler/rustc_codegen_ssa/src/mir/rvalue.rs

```diff
@@ -685,7 +685,7 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
                         let val = layout.offset_of_subfield(bx.cx(), fields.iter()).bytes();
                         bx.cx().const_usize(val)
                     }
-                    mir::NullOp::UbCheck(_) => {
+                    mir::NullOp::UbChecks => {
                         // In codegen, we want to check for language UB and library UB
```

Comment:
I don't think this comment is relevant anymore; all we're doing here is implementing the MIR intrinsic.